Fluid/Filters/ColorFilters.cs

```diff
@@ -25,6 +26,32 @@ public static FluidValue ToRgb(FluidValue input, FilterArguments arguments, Temp
             return new StringValue($"rgb({rgbColor.R}, {rgbColor.G}, {rgbColor.B})");
         }
 
+        public static FluidValue ToHex(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            var color = input.ToStringValue();
+            if (!color.StartsWith("rgb(") && color.EndsWith(")") && color.Length >= 10)
+            {
+                return NilValue.Empty;
+            }
+            
+            var rgbColor = color.Substring(4, color.Length - 5).Split(',');
```

Comment:
nit, you can use the new spread operator. Or do we need to bump the C# version in the project?

Reply:
Seems we need to bump the version

Reply:
not worth it then